aggregator/src/aggregator/aggregation_job_driver.rs

```diff
@@ -435,6 +442,7 @@ impl AggregationJobDriver {
             aggregation_job,
             &stepped_aggregations,
             report_aggregations_to_write,
+            report_ids_to_scrub,
             resp.prepare_resps(),
         )
         .await
```

Comment:
somewhat nit: Adding more functionality to `process_response_from_helper` makes me a bit uncomfortable, since it overloads that function's purpose and introduces an argument that is only necessary for `step_aggregation_job_aggregate_init`.

Can we instead scrub reports at the end of `step_aggregation_job_aggregate_init()`? Or was the intent of having it done in `process_response_from_helper` was to be concurrent with the other tx's?

Reply:
We need to scrub reports at the end of the initial step, in the same transaction that updates the aggregation state. If we scrubbed reports in an earlier transaction, we could end up succeeding at scrubbing the reports, but ultimately failing to step the aggregation job -- this would leave the aggregation job stuck, as we would no longer have the reports required to complete initialization on the next attempt.